css/main.scss

```diff
@@ -43,3 +43,14 @@ $npm-red: #C12127;
 @import "_toc.scss";
 @import "_footer.scss";
 @import "_search.scss";
+@import "_details.scss";
+@import "_readmore.scss";
+@import "_copyable.scss";
+
+// should come from bootstrap but it doesn't
+.flex-items-between {
+  -webkit-box-pack: justify;
+  -webkit-justify-content: space-between;
+      -ms-flex-pack: justify;
```

Comment:
Spacing here is a bit strange, did you mean to line it up with the other two lines?